yktracec/src/jitmodbuilder.cc

```diff
@@ -27,6 +29,8 @@ uint64_t getNewTraceIdx() {
   return TraceIdx;
 }
 
+// FIXME: A lot of these #defines should be C++ consts, then the compiler can
```

Comment:
This feels like an issue to be filed rather than a comment for this PR? [I agree with the comment FWIW!]

Reply:
Removed in Fixed in 96f196064a3dafea592a5e5a3da4c750e1a06e0a, in favour of #753